internal/test/comparator/standard.go

```diff
@@ -0,0 +1,42 @@
+package comparator
+
+import (
+	"sync/atomic"
+
+	"github.com/vdaas/vald/internal/errgroup"
+)
```

Comment:
Could you please add alias of `go-comp`.

Reply:
Sorry i am not sure what you mean 🤔

Reply:
for example.
```
type Option = cmp.Option
```

Reply:
fixed.
https://github.com/vdaas/vald/pull/602/commits/4d7920ab0518e0b39e0652da7213f0266f33922d